src/components/Pf/PfSpinner.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+type PfSpinnerProps = {
+  // none yet
+};
+
+const PfSpinner: React.SFC<PfSpinnerProps> = props => {
```

Comment:
I would like to note that this is preferred over functional components for type safety.
I didn't know about this class, just read about it

https://medium.com/@iktakahiro/react-stateless-functional-component-with-typescript-ce5043466011